config/locales/api/en.yml

```diff
@@ -20,5 +20,6 @@ en:
       mentor_request_not_found: "The mentor request was not found"
       mentor_request_locked: "This mentor request is locked by someone else."
       mentor_discussion_not_found: "The mentor discussion was not found"
-
+      mentor_discussion_not_acessible: "You do not have permission to view this mentor discussion"
```

Comment:
```suggestion
      mentor_discussion_not_accessible: "You do not have permission to view this mentor discussion"
```

Reply:
I don't understand how the tests pass with this typo.

Reply:
It's because we use the I18n.t helpers in the tests. I think we shouldn't do that, as these are integration tests for the API. I think we should assert the text directly, as it's useful information to be informed of copy changes.

Reply:
> I18n.t helpers in the tests

Yeah, but I thought the I18n helpers failed if the key was missing. If not, we should change it so they do.

I still prefer using the helpers in the tests, because I want to be able to freely change that text without changing code. Eventually the i18n will all be in a database not yml, so it should be treated as data, not code.

Reply:
Cool. Sounds good.